benchmark/benchmark.py

```diff
@@ -30,7 +30,7 @@ def prepend_pythonpath(cfg):
             prepend_pythonpath(v)
         else:
             if 'path' in k.lower():
-                cfg[k] = os.path.join(os.environ['PYTHONPATH'], v)
+                cfg[k] = os.path.join(os.environ['PYTHONPATH'].split(os.pathsep)[-1], v)
```

Comment:
If it is set to load the last one, I suggest we change all of the lines setting environment in REDME to
```bash
export PYTHONPATH=$PYTHONPATH:.. # for bash
#and for windows as well
```

Reply:
I forgot to add this setting